packages/server/lib/crons/usage.ts

```diff
@@ -0,0 +1,88 @@
+import tracer from 'dd-trace';
```

Comment:
I went for reusing the existing usage cron. I renamed it to `usage` and moved functions around but it is basically just adding a extra function to calculate billable connections